src/components/perps/BalancesTable/Columns/PnL.tsx

```diff
@@ -17,34 +20,44 @@ export const PNL_META = {
 type Props = {
   pnl: PerpsPnL
   type: PerpsPosition['type']
+  position: PerpsPosition
 }
 
 export default function PnL(props: Props) {
-  if (props.type === 'limit')
+  const { pnl, position } = props
+
+  const pnlPercentage = useMemo(() => {
+    if (!position || !position.amount || pnl.net.amount.isZero()) return BN_ZERO
+
+    const positionValue = BN(position.amount).abs().times(position.entryPrice)
```

Comment:
Rename to `entryValue`
```suggestion
    const entryValue = BN(position.amount).abs().times(position.entryPrice)
```